public/app/features/alerting/unified/components/contact-points/EditMessageTemplate.tsx

```diff
@@ -1,46 +1,46 @@
 import { RouteChildrenProps } from 'react-router-dom';
 
-import { Alert } from '@grafana/ui';
+import { Alert, LoadingPlaceholder } from '@grafana/ui';
 import { EntityNotFound } from 'app/core/components/PageNotFound/EntityNotFound';
 
-import { useAlertmanagerConfig } from '../../hooks/useAlertmanagerConfig';
 import { useAlertmanager } from '../../state/AlertmanagerContext';
-import { EditTemplateView } from '../receivers/EditTemplateView';
+import { stringifyErrorLike } from '../../utils/misc';
+import { TemplateForm } from '../receivers/TemplateForm';
+
+import { useGetNotificationTemplate } from './useNotificationTemplates';
 
 type Props = RouteChildrenProps<{ name: string }>;
 
 const EditMessageTemplate = ({ match }: Props) => {
+  const templateUid = match?.params.name ? decodeURIComponent(match?.params.name) : undefined;
```

Comment:
While we're here, we should change to `useParams` as part of the react router migration. I'll do that now